include/boost/geometry/strategies/relate/cartesian.hpp

```diff
@@ -39,7 +39,7 @@ namespace boost { namespace geometry
 namespace strategies { namespace relate
 {
 
-template <typename CalculationType = void>
+template <typename CalculationType = void, bool IsSetOperation = false>
```

Comment:
I'm against this change. If you think that set operations need different umbrella strategy then you should create another one instead of adding template argument that doesn't exist in any other umbrella strategy.

One of the reasons for umbrella strategies to exist is to have intuitive and consistent interface. This unfortunately also means: not adding arguments because it's convenient.

Reply:
Btw, what with the general one umbrella strategy `strategies::cartesian<>` that can be passed into all algorithms?
Which side strategy should be used there?
Are relational and set operations different so they need different side strategies?
Does it mean that the results of relational and set operations are not consistent?